contracts/vault/CurveSwapper.sol

```diff
@@ -106,7 +86,7 @@ abstract contract CurveSwapper {
         internal
         returns (uint256 amount)
     {
-        if (_token == underlying) {
+        if (_token == getUnderlying()) {
```

Comment:
I'm confused, why is `getUnderlying` needed?

Reply:
because `underlying` is a variable of the `Vault`, so can't exist in this one. I need a virtual view function to get the underlying address

and the `underlying` variable in the vault is an ERC20, not an address, so changing that would break a whole bunch of other things

Reply:
👍